### PR TITLE
Alex garcia lesson1 homework2

### PR DESCRIPTION
**Pull request by Saex05**
Add car project in the path ...\alex-garcia\lesson-1\homework-2-car-project

### CODE REVIEW

File: alex-garcia/lesson-1/homework-1-contact-manager/ContactsManager.java [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+    int friendsCount;
+
+    // Constructor:
+    ContactsManager(){
+        friendsCount = 0;
+        myFriends = new Contact[500];
+    }
+
+    // Methods:
+    void addContact(Contact Contact){
```
Comment (J4rv1sX) [inline alex-garcia/lesson-1/homework-1-contact-manager/ContactsManager.java]: Please start parameters with lower case

File: alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    private int automobileCount;
+
+    //Constructor;
+    CarBehavior(){
+        automobile = new Car[3];
+        automobileCount = 0;
+    }
+
+    // Methods:
+    void addCar(Car Car){
```
Comment (J4rv1sX) [inline alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java]: Same here, start with lowercase for Fields, Methods and Parameters

File: alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
+//https://vehiculos.doncomos.com/calcular-consumo-gasolina
+
+class CarBehavior {
+    static class Car {
+        String name;
```
Comment (J4rv1sX) [inline alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java]: I will recommend to just to have a single class called Car, in either case please define your Fields with private access modifier and create access methods is necessary (getters and setters).

**Review by J4rv1sX — CHANGES_REQUESTED** [review]

File: alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java (hunk 2 above)
Comment (Saex05) [reply alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java]: Update: Parameters started with lower case

File: alex-garcia/lesson-1/homework-1-contact-manager/ContactsManager.java (hunk 1 above)
Comment (Saex05) [reply alex-garcia/lesson-1/homework-1-contact-manager/ContactsManager.java]: Update: Parameters started with lower case

File: alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java (hunk 3 above)
Comment (Saex05) [reply alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java]: Update: Fields defined  with private access modifier and access methods (getters) created in the car project.

**Pull request closed without merging**